jenkins/pipelines/cd/nightly/tiup-release-tidb-master-nightly.groovy

```diff
@@ -218,16 +218,52 @@ retry(2) {
                 parallel builds
             }
 
-            RELEASE_TAG = "nightly"
 
-            stage("TiUP build") {
-                def job = build job: "tiup-mirror-online-ga",
+            stage("Publish"){
+                def NIGHTLY_RELEASE_TAG = 'nightly'
+                jobs = [:]
+                jobs["tiup"] = {
+                    def job = build job: "tiup-mirror-online-ga",
                         wait: true,
                         parameters: [
-                                [$class: 'StringParameterValue', name: 'RELEASE_TAG', value: RELEASE_TAG],
+                                [$class: 'StringParameterValue', name: 'RELEASE_TAG', value: NIGHTLY_RELEASE_TAG],
                                 [$class: 'StringParameterValue', name: 'TIUP_ENV', value: "prod"],
                         ]
-                PRODUCED_VERSION = job.getBuildVariables().PRODUCED_VERSION
+                    PRODUCED_VERSION = job.getBuildVariables().PRODUCED_VERSION
+                }
+                jobs["docker"] = {
+                    build job: "community-docker-multi-products",
+                        parameters: [
+                            string(name: 'RELEASE_BRANCH', value: 'master'),
+                            string(name: 'RELEASE_TAG', value: RELEASE_TAG),
+                            booleanParam(name: 'FORCE_REBUILD', value: false),
+                            booleanParam(name: 'NEED_DEBUG_IMAGE', value: false),
+                            string(name: 'TIDB_HASH', value: tidb_sha1),
+                            string(name: 'TIKV_HASH', value: tikv_sha1),
+                            string(name: 'PD_HASH', value: pd_sha1),
+                            string(name: 'TIFLASH_HASH', value: tiflash_sha1),
+                            string(name: 'NG_MONITORING_HASH', value: ng_monitoring_sha1),
+                            string(name: 'TIDB_BINLOG_HASH', value: tidb_binlog_sha1),
+                            string(name: 'TICDC_HASH', value: cdc_sha1),
+                            string(name: 'IMAGE_TAG', value: NIGHTLY_RELEASE_TAG),
+                            string(name: 'HUB_PROJECT', value: 'rc'),
```

Comment:
How about push nightly image to `qa` project instead of `rc` project

Reply:
yes

Reply:
ok